### PR TITLE
Update hytteforeningen.md (new board info)

### PR DESCRIPTION
**Pull request by einmid**
Updated HF new board positions

### Changes

- Update board info

### CODE REVIEW

File: frontend/content/organizations/hytteforeningen.md [hunk 1]
```diff
@@ -5,29 +5,30 @@ logo: /static/cabins/logo.svg
...
-    phoneNumber: 457 75 516
+  # log:
+    name: 
+    phoneNumber: 
     title: Logistikkansvarlig
-  pr:
-    name: Ingrid Bergan
-    phoneNumber: 919 14 214
-    title: PR- og sosialansvarlig
+
```
Comment (larwaa) [inline frontend/content/organizations/hytteforeningen.md:31]: If these are no longer in use, just remove them entirely. 😊

**Review by larwaa — CHANGES_REQUESTED** [review]
I love to see this, want to inspire others to take the same kind of ownership? 🙏 I'll address the build issues and then get this merged!

File: frontend/content/organizations/hytteforeningen.md (hunk 1 above)
Comment (einmid) [reply frontend/content/organizations/hytteforeningen.md:31]: Makes sense, tricked by syntax highlighting.

The positions will be filled in August. Commenting out the lines was done to make it easier/quicker to update info this fall. Is that a fair way of doing it, or should I remove the lines?

**Review by larwaa — APPROVED** [review]
LGTM

**Pull request merged**